quickstarts/uppercase-firestore/README.md

```diff
@@ -1,19 +1,64 @@
 # Firebase SDK for Cloud Functions Quickstart - Cloud Firestore
 
-This quickstart demonstrates using **Firebase SDK for Cloud Functions** setup
-with a **Cloud Firestore**.
+This quickstart demonstrates using the **Firebase SDK for Cloud Functions** with
+**Cloud Firestore**.
 
 ## Introduction
 
 This sample app does two things:
 
-- Create messages in Cloud Firestore using a simple HTTPS request which is
+- Creates messages in Cloud Firestore using a simple HTTPS request which is
   handled by an HTTP function. Writing to Cloud Firestore is done using the
   Firebase Admin SDK.
 - When a message gets added in the Cloud Firestore, a function triggers and
```

Comment:
Suggest omitting "the". (just "in Cloud Firestore")

Reply:
removed